paasta_tools/mesos_tools.py

```diff
@@ -468,10 +469,15 @@ def get_mesos_state_from_leader():
     return state
 
 
-def get_mesos_quorum(state):
+def get_mesos_state_summary_from_leader():
+    res = master.CURRENT.fetch("/master/state-summary")
```

Comment:
@solarkennedy where do you see that? I'm looking at [these docs](https://mesos.apache.org/documentation/latest/endpoints/master/state-summary/)